modules/oxide/src/less/theme/components/color-swatch/color-swatch.less

```diff
@@ -23,6 +23,11 @@
   }
 
   .tox-swatches__row {
+
+    @media (forced-colors: active) {
+      forced-color-adjust: none;
```

Comment:
That defeats the purpose of high contrast mode?

In the past we've just replaced all icons with the tooltip contents to work around problems with contrast.

Reply:
what about `colourPicker`?  That reminds me maybe I missed to update that one.

Reply:
I think showing the text for each colour will be essential, since high contrast generally doesn't provide a way to show actual colours. Particularly for background colour.